aspnetcore/includes/net-core-prereqs-vsc-3.0.md

```diff
@@ -1,5 +1,5 @@
 * [Visual Studio Code](https://code.visualstudio.com/download)
-* [.NET Core SDK 3.0](https://dotnet.microsoft.com/download/dotnet-core/3.0)
 * [C# for Visual Studio Code (latest version)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
+* [!INCLUDE [](~/includes/3.0-SDK.md)]
```

Comment:
There's a formatting quirk in that there's a bit of extra padding (a `p` tag) between the bullets when using an include like this. It all looks fine if the include is the last bullet in the list. I changed the other VS Code instructions to match the order change here.